ts/components/services/TosAndPrivacyBox.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+
+import { TranslationKeys } from "../../../locales/locales";
+import I18n from "../../i18n";
+import { handleItemOnPress } from "../../utils/url";
+
+import IconFont from "../ui/IconFont";
+import { H3 } from "../core/typography/H3";
+import { Link } from "../core/typography/Link";
+import { IOColors } from "../core/variables/IOColors";
+import ItemSeparatorComponent from "../ItemSeparatorComponent";
+
+const styles = StyleSheet.create({
+  infoItem: {
+    paddingVertical: 16,
+    flex: 1
+  },
+  header: {
+    flexDirection: "row",
+    paddingVertical: 8
+  },
+  icon: {
+    marginRight: 3.85
+  }
+});
+
+type Props = {
+  privacyUrl?: string;
+  tosUrl?: string;
+};
+
+/**
+ * Renders a dedicated section with TOS, Privacy urls, and the header.
+ */
+const TosAndPrivacy: React.FC<Props> = ({ tosUrl, privacyUrl }) => (
+  <View>
+    <View style={styles.header}>
+      <IconFont
+        name={"io-lucchetto"}
+        color={IOColors.bluegrey}
+        style={styles.icon}
+      />
+      <H3 weight={"SemiBold"} color={"bluegrey"}>
+        {I18n.t("services.tosAndPrivacy")}
+      </H3>
+    </View>
+    {tosUrl && (
+      <>
+        {renderLinkRow("services.tosLink", tosUrl)}
+        <ItemSeparatorComponent noPadded />
```

Comment:
maybe this can be moved inside `renderLinkRow`?

Reply:
Refactored [here](https://github.com/pagopa/io-app/pull/3200/commits/3ef427e3ae046af2cb180cc1a0fa231a59c5a767#diff-a6c4bb541039dee54bf32f6b1bb41ed1970b3b733f22099e3cb609b9f8c528b4R45) and indeed looks much cleaner!